terraform/files/bin/create_cluster.sh

```diff
@@ -35,6 +35,9 @@ fi
 CCCFG="$HOME/${CLUSTER_NAME}/clusterctl.yaml"
 fixup_k8s_version.sh $CCCFG || exit 1
 
+kubectl config use-context kind-kind || exit 1
+kubectl create namespace $CLUSTER_NAME || exit 1
```

Comment:
Will it work, when I rerun this script a second time and the namespace already exists?

Reply:
Reworked completly